front-end/virtual_pet/lib/pages/home.dart

```diff
@@ -19,77 +19,79 @@ class HomeState extends State<Home> {
-        bottomSheet: Container(
-            padding: const EdgeInsets.only(bottom: 10.0),
-            color: Theme.of(context).canvasColor,
-            width: MediaQuery.of(context).size.width,
-            height: 60.0,
-            child: Row(
-              mainAxisAlignment: MainAxisAlignment.spaceEvenly,
-              children: [
-                IconButton(
-                  onPressed: () {
-                    /// when pressed, the push function will be called for UI to display page two
-                    Navigator.push(
-                      context,
-                      MaterialPageRoute(builder: (context) => const ToDoListPage()),
-                    );
-                  },
-                  icon: const Icon(Icons.event_available, size: 50.0),
-                ),
-                IconButton(
-                  onPressed: () {
-                    /// when pressed, the push function will be called for UI to display pomodoro page
-                    Navigator.push(
-                      context,
-                      MaterialPageRoute(
-                          builder: (context) => const PomodoroPage()),
-                    );
-                  },
-                  icon: const Icon(Icons.timer, size: 50.0),
-                ),
-                IconButton(
-                  onPressed: () {
-                    /// when pressed on button, the app will pop up a task-complete window
-                    Navigator.push(
-                        context,
-                        MaterialPageRoute(
-                          builder: (context) => const TaskComplete(),
-                        ));
-                  },
-                  icon: const Icon(Icons.text_fields, size: 50.0),
-                ),
-                IconButton(
-                  onPressed: () {
-                    /// when pressed, the push function will be called for UI to display settings menu
-                    Navigator.push(
-                      context,
-                      MaterialPageRoute(
-                          builder: (context) => const SettingsPage()),
-                    );
-                  },
-                  icon: const Icon(Icons.settings, size: 50.0),
-                ),
-                IconButton(
-                  onPressed: () {
-                    /// when pressed, the push function will be called for UI to display store menu
-                    Navigator.push(
-                      context,
-                      MaterialPageRoute(
-                          builder: (context) => const StorePage()),
-                    );
-                  },
-                  icon: const Icon(Icons.store, size: 50.0),
-                ),
-              ],
-            )
+      /// instead of using bottom sheet, we could also try bottomNavigationBar
+      bottomSheet: Container(
+        padding: const EdgeInsets.only(bottom: 10.0),
+        color: Theme.of(context).bottomSheetTheme.backgroundColor,
```

Comment:
This setting of the `color` property can also be removed for a similar reason, but owing to the fact that the setting of the `theme` property in `main.dart` is what is actually causing the bottom bar to become green.